libnetwork/iptables/firewalld_test.go

```diff
@@ -17,10 +17,17 @@ func skipIfNoFirewalld(t *testing.T) {
 		t.Skipf("cannot connect to D-bus system bus: %v", err)
 	}
 	defer conn.Close()
+	currentState, err := connection.sysObj.GetProperty(dbusInterface + ".state")
```

Comment:
I'm curious why this uses a different API as above, which uses this;


```go
connection.sysObj.CallWithContext(ctx, "org.freedesktop.DBus.Properties.Get", 0, dbusInterface, "state").Store(&currentState)
```

Or is that intentional to compare things?